### PR TITLE
update org description, remove client section, fix partner images

### PR DESCRIPTION
**Pull request by citizendrummer**
- update organization description

- remove client section

- fix partner image styles

- remove sitecake files and markup

https://preview.c9users.io/citizendrummer/cs-new-mission/index.html

### CODE REVIEW

**Comment by citizendrummer** [comment]
@arelia @DerekTownsend @briberri could someone review these changes then merge? Thanks!

**Comment by arelia** [comment]
@citizendrummer good job with the clean up. For future pull requests, it might be easier to review if you break the changes up into separate commits or even separate PRs. That way, if anything breaks it's easier to determine which change caused bugs. It would be easier to review and approve the Sitecake changes because the PR would only include `sitecake/` files and `.sc-` classes

**Comment by arelia** [comment]
@citizendrummer Also, include a link to a test site so that we can see how the changes will affect the site, like @jabari-austin did here: https://github.com/CoderSpace/coderspace2/pull/3

File: styles/style.css [hunk 1]
```diff
@@ -8,13 +8,10 @@
 }
 
 .img-partners {
-    display:inline-block;  
-    margin:25px;
+    display: block;
```
Comment (arelia) [inline styles/style.css]: this is the type of change I'd want to see on a test site first, will you please add one?

**Comment by citizendrummer** [comment]
Thanks @arelia. I updated the PR with link to test site.

Do you want me to redo all the changes separately?

**Pull request closed without merging**